### ISSUE TITLE
Error: Patch file not found... during install

### ISSUE THREAD

**Issue by mickeyvip** [underfin/vite-plugin-vue2#26]
Hello and thank you for the plugin.

I am installing it and, while all seems to work, I see the error:

```
> vite-plugin-vue2@1.1.1 postinstall /Users/michael/Projects/Vue/vite-vue2/node_modules/vite-plugin-vue2
> patch-package

patch-package 6.2.2
Applying patches...
Error: Patch file found for package component-compiler-utils which is not present at node_modules/@vue/component-compiler-utils
```

![image](https://user-images.githubusercontent.com/220427/105609889-ef95cb00-5db4-11eb-8032-65d44929347b.png)

Am I doing something wrong?

**Comment by underfin** [comment]
Hmmm. This is a bad-fix for alias asset https://github.com/underfin/vite-plugin-vue2/issues/21. I already open a pr at component-compiler-utils to fix this.

**Comment by mickeyvip** [comment]
when will `1.2.0` be published to npm? I still see there is `1.1.1` there.

**Comment by underfin** [comment]
Sorry for missed publish error... The 1.2.0 is already published.
